### DISCUSSION TITLE
Windows版安装包有可能支持较低版本Windows10吗？

### DISCUSSION THREAD

**Discussion by AimerEGOIST** [xiaoyaocz/flutter_dmzj#161] [General]
手里有台wp，以前大佬做的uwp版dmzj因为api改了没法用了，这个新版的安装包wp没办法识别。Win10版本是1709，10.0.15254.603，如果不行就算了😂只是想让wp发挥下余热

**Comment by xiaoyaocz** [comment]
不行，Windows版不是UWP

**Reply by AimerEGOIST** [reply]
是吗，那太可惜了😔😔
